azure-pipelines.sds.pr-release.yml

```diff
@@ -105,6 +105,7 @@ stages:
         displayName: "Build and Push Image ${{ variables.git_repo_name }}"
         steps:
           - checkout: self
+            fetchTags: true
```

Comment:
add `fetchTags: true` to the master pipeline too for the same job

Reply:
done